api/client.go

```diff
@@ -172,6 +172,17 @@ func WithTimeout(timeout time.Duration) Option {
 	})
 }
 
+// WithTransport changes the default transport to increase TLSHandshakeTimeout
+func WithTransport() Option {
+	transport := &http.Transport{
+		TLSHandshakeTimeout: 60 * time.Second,
```

Comment:
should we increase this timeout by default? I would say so but I don't know why we wouldn't want that? 🤔